hbw/config/locales/en.yml

```diff
@@ -2,6 +2,8 @@ en:
   hbw:
     js:
       submit: Submit
+      delete: Cancel task
```

Comment:
Please don't forget to add russian translation, because without it JS will die then rendering this form with russian locale.